# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/pspresent/install.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-03-03 01:13:39.952071248 +0100
+++ ./Makefile	2004-03-03 01:14:44.590244752 +0100
@@ -21,6 +21,10 @@
 $(TARGET): $(OBJS)
 	$(CC) -o $(TARGET) $(OBJS) $(LDLIBS)
 
+install:
+	install $(TARGET) /usr/bin/$(TARGET)
+	cp pspresent.1 /usr/man/man1/
+
 clean:
 	rm $(TARGET) $(OBJS)
 
